serializers/src/glTF/2.0/babylon.glTFMaterialExporter.ts

```diff
@@ -501,20 +501,8 @@ module BABYLON.GLTF2.Exporter {
                     // Read data from WebGL
                     const canvas = engine.getRenderingCanvas();
                     if (canvas) {
-                        BABYLON.Tools.ToBlob(canvas, blob => {
-                            if (blob) {
-                                let fileReader = new FileReader();
-                                fileReader.onload = (event: any) => {
-                                    let base64String = event.target.result as string;
-                                    hostingScene.dispose();
-                                    resolve(base64String);
-                                };
-                                fileReader.readAsDataURL(blob);
-                            }
-                            else {
-                                reject("gltfMaterialExporter: Failed to get blob from image canvas!");
-                            }
-                        });
+                        const dataURL = canvas.toDataURL();

```

Comment:
You still need to do the toblob if available to not lose color when alpha value is low on compatible browsers.

Reply:
This could be your fallback if toblob is not available.